core/metadata.js

```diff
@@ -72,11 +72,35 @@ const CONFLICT_REGEXP = new RegExp(
   `-conflict-${DATE_REGEXP}${SEPARATOR_REGEXP}`
 )
 
+const LOCAL_ATTRIBUTES = [
+  'md5sum',
+  'class',
+  'docType',
+  'executable',
+  'updated_at',
+  'mime',
+  'size',
+  'ino',
+  'fileid'
```

Comment:
What is the purpose of keeping `ino` and `fileid` here? In which case they will be different to the `ino` and `fileid` on the global document?

Reply:
They're probably not gonna be different but the goal is to move away from using the attributes of the global document so I'm adding them now.